services/webnn/ort/context_impl_ort.cc

```diff
@@ -165,7 +191,7 @@ SessionOptions::Create(const mojom::CreateContextOptions::Device device_type) {
     // backend.
     // https://onnxruntime.ai/docs/execution-providers/OpenVINO-ExecutionProvider.html#other-configuration-settings
     CALL_ORT_FUNC(ort_api->SetSessionGraphOptimizationLevel(
-        session_options.get(), GraphOptimizationLevel::ORT_DISABLE_ALL));
+        session_options.get(), kGraphOptimizationLevelParam.Get()));
```

Comment:
I would recommend to let each EP has default optimization level setting, at leas for OpenVINO EP, its doc recommends to disable all. Then developers can use this feature flag to override it. WDYT?

Reply:
It's not easy to set different default level for EPs, unless we introduce three feature name like: `kGraphOptimizationLevelOvParam, kGraphOptimizationLevelCpuParam, kGraphOptimizationLevelDmlParam`, wdyt?

Reply:
I meant set the different default in code, the feature flag only override the overall optimization level at runtime. Does it make sense?

Reply:
`kGraphOptimizationLevelParam.Get()` will return the default value even if you didn't pass the flag

Reply:
Can I disable this feature?

Reply:
Do you mean someting like:
```
ov_level = disable
if ConfigureWebNNOrt enabled:
ov_level = kGraphOptimizationLevelParam.Get()
```

Reply:
Use command line flag now